src/System.Reflection.Context/src/System/Reflection/Context/Projection/IProjectable.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Reflection.Context.Projection
+{
+    // TODO: Temporary interface until we can add ReflectionContext to Type/Assembly
```

Comment:
Is this TODO stale?  Can it be deleted?  Or is there an issue number associated with it that can be added here?